rabbitmq/assets/configuration/spec.yaml

```diff
@@ -127,10 +127,28 @@ files:
         example:
           - <VHOST_NAME_1>
           - <VHOST_NAME_2>
-    - template: instances/http
+    - name: prometheus_plugin
+      description: |
+        Settings for talking to the prometheus plugin of rabbitmq.
+        Specify this section to trigger metric collection from the prometheus plugin endpoints.
+      options:
+        - name: url
+          required: true
+          description: |
+            Base URL for openmetrics endpoints.
+          value:
+            type: string
+            example: 'http://localhost:15692'
+        - name: include_aggregated_endpoint
```

Comment:
@fanny-jiang see [here](https://datadoghq.atlassian.net/wiki/spaces/~6334a86f8b75455be456d0c9/pages/2753036726/Implementation+Plan#Option-E%3A-Separate-aggregated-from-unaggregated-endpoints) for motivation.